packages/vm/src/imports.rs

```diff
@@ -224,9 +224,6 @@ mod test {
     static INIT_AMOUNT: u128 = 500;
     static INIT_DENOM: &str = "TOKEN";
 
-    #[cfg(feature = "singlepass")]
-    use crate::backends::singlepass::GAS_LIMIT;
-    #[cfg(not(feature = "singlepass"))]
     const GAS_LIMIT: u64 = 10_000_000_000;
```

Comment:
what is this used for, then?

Reply:
The updated versin of the commit probably makes clear, that `GAS_LIMIT` and `MAX_GAS_LIMIT` are separate things: https://github.com/CosmWasm/cosmwasm/pull/470/commits/22e8a9b1ff16576c5f63e47d5a34300421a4d5fd

Reply:
oh i didn't notice that this is in the tests. all's good :)

Reply:
Glad we talked about it. Now that `GAS_LIMIT` was renamed to `MAX_GAS_LIMIT`, this confusion should not happen easily again in the future.